tests/test_update_api_key.py

```
@@ -366,6 +342,77 @@ def test_update_api_key_skips_blank_env_lines(self, tmp_path, fake_crypto):
         assert "" in lines  # blank line preserved
         assert any(line.startswith("OPENAI_API_KEY=encrypted:") for line in lines)
 
+    def test_update_api_key_supports_free_profile(self, tmp_path, fake_crypto):
```

Ruff (0.13.2)

Unused method argument: `fake_crypto`

(ARG002)